src/wp-includes/class-wp-block-bindings-registry.php

```diff
@@ -32,6 +32,19 @@ final class WP_Block_Bindings_Registry {
 	 */
 	private static $instance = null;
 
+	/**
+	 * Allowed block that can use the block bindings API.
```

Comment:
Nitpick: block**s** (plural)
```suggestion
	 * Allowed blocks that can use the block bindings API.
```